cmd/app.go

```diff
@@ -815,6 +917,23 @@ func parseExternalID(e string) (map[string]string, error) {
 	return parsed, nil
 }
 
+// parseChartInfo gives back is local folder deployment or not and a map with the keys catalog, template and version
+func parseChartInfo(c string) (bool, map[string]string, error) {
```

Comment:
This function has no reference anywhere.

Reply:
Removed